src/appsettings.Development.json

```diff
@@ -6,5 +6,8 @@
       "System": "Information",
       "Microsoft": "Information"
     }
+  },
+  "ApiConnection": {
+    "Uri": "http://search-and-compare-api-bat-development.e4ff.pro-eu-west-1.openshiftapps.com/api/"
```

Comment:
Shouldnt this be `https://bat-dev-search-and-compare-api-app.azurewebsites.net/`.

Openshift is dead

Reply:
I knew something didn't seem right when I copied it from the other config file. I'd better update both of them!